firebase-installations/src/main/java/com/google/firebase/installations/local/IidStore.java

```diff
@@ -44,9 +44,9 @@
   private static final String STORE_KEY_PUB = "|S||P|";
   private static final String STORE_KEY_ID = "|S|id";
   private static final String STORE_KEY_TOKEN = "|T|";
-  private static final String DEFAULT_SCOPE = "|*";
   private static final String JSON_TOKEN_KEY = "token";
   private static final String JSON_ENCODED_PREFIX = "{";
+  private static final String [] ALLOWABLE_SCOPES = new String[]{"*", "FCM", "GCM"};
```

Comment:
Is the space on purpose between String and []?

Reply:
no, done